app/src/features/projects/view/components/GeneralInformation.test.tsx

```diff
@@ -51,7 +87,44 @@ describe('GeneralInformation', () => {
     fireEvent.click(getByText('Save Changes'));
 
     await waitFor(() => {
-      expect(history.location.pathname).toEqual(`/projects/${getProjectForViewResponse.id}/details`);
+      expect(mockBiohubApi().project.updateProject).toHaveBeenCalledTimes(1);
+      expect(mockBiohubApi().project.updateProject).toBeCalledWith(getProjectForViewResponse.id, {
+        project: {
+          project_name: 'project name',
+          project_type: 1,
+          project_activities: [1, 2],
+          climate_change_initiatives: [1, 2, 3],
+          start_date: '2020-04-20',
+          end_date: '2020-05-20',
+          revision_count: 2
+        }
+      });
+
+      expect(mockRefresh).toBeCalledTimes(1);
+    });
+  });
+
+  it('displays an error dialog when fetching the update data fails', async () => {
```

Comment:
can we add the tests that I added yesterday to this component as well so it tests the `onClose` callback as well and the apierror stuff

Reply:
yup!